homeassistant/components/ecobee/repairs.py

```diff
@@ -0,0 +1,66 @@
+"""Repairs support for Ecobee."""
```

Comment:
I am doubting if this is something that we should solve globally instead of creating repairs for every separate integration, since we have 68 core integration that need to have this fixed

Reply:
The point is that when we raise an issue we need ensure the user has migrated. We do this with yaml by checking the config has been removed. Be cause ecobee already has an entry we cannnot do this.
As an alternative we register an issue if the user uses the old service. But I think that is not enough.
So the repair ensures the user is not using the old service anymore.
We could have a shared translation key with placeholders though.

Reply:
After some discussion we have decided to only raise an issue if the old service is actively being used. The old service is not removed yet. At this point it should be enought to inform users to migratie their automations. When we are at a point that we want do deprecate the legacy service globally we will raise the issue the moment the platform is loaded.